src/login/logind-core.c

```diff
@@ -71,6 +71,8 @@ void manager_reset_config(Manager *m) {
 
         m->kill_only_users = strv_free(m->kill_only_users);
         m->kill_exclude_users = strv_free(m->kill_exclude_users);
+
+        m->stop_idle_session_usec = 0;
```

Comment:
USEC_INFINITY sounds like the better value to mean "never" here

0 conceptually means "immediately" more.

Reply:
Ok, but I will use `config_parse_sec_fix_0` as a parser. I really want 0 to mean infinity, i.e. timeout disabled. Setting this to 0 doesn't make sense as you would be esseantially saying that you want session to be stop immediately after it started.

Reply:
well, it's ok to use the parser that accepts 0, i don't mind, but I think we shold document "infinity" as a way to turn this off, and internally encode it like that...